patterns/mixed/variables.tf

```
@@ -592,3 +592,15 @@ variable "IC_SCHEMATICS_WORKSPACE_ID" {
 }
 
 ##############################################################################
+
+
+##############################################################################
+# CBR variables
+##############################################################################
+variable "existing_vpc_cbr_zone_id" {
+  type        = string
+  description = "Existing VPC CBR zone id. This is to open traffic flow only from landing zone VPCs to specific cloud services for an existing CBR rule that has context (network zones) set to VPC network zone."
```

```suggestion
  description = "The existing VPC network zone id for a context-based restriction rule with context (network zones) set to the VPC network zone. The rule allows traffic to flow only from the landing zone VPCs to specific cloud services."
```

I don't think there's really anything called a CBR zone, right? You use the command to create network zones. Not sure if I've captured the intent accurately though. For example, I'm not sure what `This` refers to in the original description.